TACC/Core-Portal
# task/FP-1901: Replace system monitor endpoint with TAP

## Overview
Change endpoints for System Monitor from TUP to the TAP status server located here [https://tap.tacc.utexas.edu/status/](https://tap.tacc.utexas.edu/status/).


## Related

* [FP-1901](https://jira.tacc.utexas.edu/browse/FP-1901)

## Changes

- Updated `SYSTEM_MONITOR_DISPLAY_LIST` settings_default to now match the key of the TAP response. Updated URL in server settings to point to TAP status server.  Updated system_monitor view to return a response as close to the TUP response as possible with the new data so as not to interfere with the CMS Plug in.

- Updated fixtures to be real responses from TAP system status. 

- Adjusted tests and views to reflect what TAP response includes and adjusting to still create job object and load_percentage for CMS plugin

## Testing

1. Open https://tap.tacc.utexas.edu/status/
2. Open cep.test and  go to dahsboard.
3. Confirm the system monitor shows waiting, running, and operational/maintenance are correctly cross checking data for jobs and loading with response here [https://tap.tacc.utexas.edu/status/](https://tap.tacc.utexas.edu/status/) to make sure system monitor shows the correct values. 
*** Note - may need to refresh https://tap.tacc.utexas.edu/status/ and your dashboard at the same time to make sure the numbers are the same since it's updated live.

## UI
No UI changes
